src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggBehandlingPåVent/SettEllerOppdaterVenting.tsx

```diff
@@ -27,7 +28,10 @@ const SettEllerOppdaterVenting: React.FC<IProps> = ({ behandling }) => {
             onSubmit(
                 {
                     method: erBehandlingAlleredePåVent ? 'PUT' : 'POST',
-                    data: { frist: skjema.felter.frist.verdi, årsak: skjema.felter.årsak.verdi },
+                    data: {
+                        frist: formatterDateTilIsoString(skjema.felter.frist.verdi),
+                        årsak: skjema.felter.årsak.verdi,
+                    },
```

Comment:
Her er ikke data-objektet typet. Noe som gjør at man her kan sende inn frist=undefined og årsak=undefined (skjer aldri i praksis pga validering tho). Men backend er ikke nullable. Har lyst til å fikse det, men usikker på hvordan det er best å gjøre det. Enten la backend være nullable, eller kaste feil på en eller annen måte her om frist eller årsak = undefined?

Reply:
Lag gjerne en type i frontend for restobjektet der feltene ikke er nullable. Får vi gitt feilmelding her på en god måte?

Reply:
`kanSendeSkjema()` som kalles over her sjekker at begge feltene har verdi (altså ikke er undefined), så man burde egentlig aldri kunne komme ned hit med ugyldige verdier. Men det skjønner ikke editoren 😅 funker jo åpenbart hvis man utvider if-en, men det ser ikke så veldig pent ut da. Men klarer ikke komme på en bedre løsning... 

<img width="777" alt="image" src="https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/3bcaf1ec-8339-4c79-8037-ab8060f1f52d">

Reply:
Avklart med @daphneleebeek, går for en løsning som det på bildet her